STYLE-GUIDE.md

```diff
@@ -2,13 +2,54 @@
+These rules should be followed in all exercises. 
+Existing descriptions and test-cases can be updated to adhere to them without requiring "replacement" test cases.
+
+### Consistency within an exercise.
+
+There are some terms that have multiple valid spellings (e.g. "lower case" vs "lowercase"). 
+Where a consistent style has not been agreed within this document, these must be consistent within an exercise.
+
+### Exceptions
+
+Exercises may vary from these rules if there is general consensus across maintainers that this is reasonable. 
+For example, an exercise about converting between different units might choose not to use SI measurements.
+
+## Language
+
+All content should be written in US English. In the future other translations may occur, but the "official" Exercism language is US English.
+
+### Measurements
 All units of measurement must be [SI](https://en.wikipedia.org/wiki/International_System_of_Units) or [SI-derived](https://en.wikipedia.org/wiki/SI_derived_unit) units.
-In the future other translations may occur, but the "official" Exercism language is US English.
 
-## Explain or substitute mathematical and esoteric terms.
+### Abbreviations, acronyms and initialisms
+
+Abbreviations, acronyms and initialisms are often more difficult to understand than other phrasing options, and can alienate people trying to learn.
+
+Many abbreviations are jargon. Avoid jargon where possible by using alternative language. Don't use abbreviations unless either:
+- The abbreviated term is better known than the unabbreviated term
+- The text will be excessively verbose without abbreviation
+
+When using abbreviations always explain what the term means on its first use. This will often, but not always, include expanding the abbreviation. It will rarely be sufficient to only expand the abbreviation.
+
+Here are some example rules:
+- Prefer "if I recall correctly" to "IIRC"
+- Prefer "as far as I know" to "AFAIK"
+- Prefer "queue" to "FIFO" and "stack" to "FILO"
+- Instead of describing an interface as "RESTful", describe it's specific properties
+- If you must use CRUD, explain that it stands for Create, Read, Update, Delete and that these are the basic SQL actions
+
+An some examples of good usage:
+- "HyperText Markup Language (HTML) is the language used to describe document structure and content on the web" _(expanded and explained)_
+- "DNA, a set of chemical instructions that influence how our bodies are constructed" _(DNA is not expanded because "deoxyribonucleic acid" is unlikely to help explain what DNA is to our audience)_
+- "NASA, the United States' space agency, launched the Mariner 2 space probe in..." _(NASA is not expanded because the "National Aerospace and Space Administration" is much better known by its acronym than by its expanded name)_
+- "The Department of Motor Vehicles (DMV) is filled with sloths. That's why everything takes forever at the DMV" _(Define DMV the first time it is used)_
+
```

Comment:
The current rules forbid uses of "e.g.", "docs", "etc." that don't also define those terms. That's okay if we're sure it's what we want.

Reply:
Right. `etc.` `e.g.` and `ie.` id' like to have as _exceptions_.

Reply:
What about contractions?

- I'd vs I would
- won't vs would not
- couldn't vs could not

Reply:
won't is short for "will not" :)

I think contractions are fine and don't fall under this rule, but we could add a rule that they're fine?

Reply:
Yeah, contractions in général are okay I think.

Reply:
But sometimes hard to get right.  Regionally (to some ill-conceived idea of "region"), "I'd" might be "I could" or "I would" or "I should" but how do you know which is which as a non-native speaker?  And of course the "would" or "will" problem with "won't".  It is just as easy to write carefully and avoid the contraction for the reader.  (Burden on the writer).  The same way we are careful in how we write e-mail, and don't top post, because it is easier for the writer, but harder on the reader.  (Our readers are kind, we should be kind as well.)

Reply:
I don't know of any regional dialect that interprets won't as would not or I'd as anything but "I would" 🤷 

I don't want to have to write out don't and won't without contractions all the time, that would feel and (imo) read weird; as if I was putting on an accent.

Reply:
Don't won't can't shouldn't couldn't wouldn't are probably all fine and more well known.

Shan't mayn't mightn't mustn't oughtn't needn't and daren't probably are not.

Reply:
Some of those are pretty common where I am, and some are even more common where I have been.  For writing, I avoid them, because I know I write normally to a more "global" audience.

Reply:
What I meant is that the top list, together with the following:

- that's 
- there's
- what's 
- we're 
- there's
- they're
- that's 
- he'll 
- she'll 
- you'll 
- I'm, 
- I've, 
- let's 
- isn't 
- aren't

...are taught as common contraction, according to my recollection, and the bottom ones aren't. In some of these cases, they might even be more well known than the un-contracted form, but I have no data on that. Just a feeling.

I really don't want to disallow _contractions_, but IF we will disallow them, I would favour a list that's only the uncommon contractions.

Reply:
I try to avoid all of them, but yes, the common ones, when I write, I generally write in my "voice" and so they do at times sneak in.  Maybe we should state "disfavor" contractions, but not outright disallow 'em.

Reply:
Good discussion - thanks. 

I've added/merged a suggestion above - pls check it over. I don't think we need contractions generally in exercises, but I'm not going to restrict them in mentoring as I want people to be able to use their voice.